src/plugins/data/public/search/aggs/buckets/lib/time_buckets/time_buckets.ts

```diff
@@ -17,7 +17,10 @@
  * under the License.
  */
 
-import { isString, isObject as isObjectLodash, isPlainObject, sortBy } from 'lodash';
+import isString from 'lodash/isString';
+import isObject from 'lodash/isObject';
```

Comment:
This should be `isObjectLodash`

Reply:
Added as a suggestion to make it easy to apply.